subcommand/injector/command.go

```diff
@@ -291,15 +306,28 @@ func (c *Command) certWatcher(ctx context.Context, ch <-chan cert.Bundle, client
 		if isLeader && c.flagAutoName != "" && len(bundle.CACert) > 0 {
 			// The CA Bundle value must be base64 encoded
 			value := base64.StdEncoding.EncodeToString(bundle.CACert)
+			payload := fmt.Sprintf(
```

Comment:
Wonder if we should make `payload` a byte slice to avoid the duplicated conversion below.

Reply:
Done in 91474cf92a00037fb2e66cbf76359412819ee412.